cli/runtime/apis/cli/v1alpha1/catalog_types.go

```diff
@@ -141,6 +141,10 @@ type PluginDescriptor struct {
 	// E.g., cluster/v0.3.2@sha256:...
 	InstallationPath string `json:"installationPath"`
 
+	// TestPluginInstallationPath is a installation path for a test plugin binary.
+	// E.g., cluster/test-v0.3.2
+	TestPluginInstallationPath string `json:"testPluginInstallationPath"`
```

Comment:
Why do we need this to be in the `PluginDescriptor`?  The `TestPluginInstallationPath` is built using a fixed pattern so do we really need to store it like this instead of building it when we need it?  Could we instead adjust `runner.RunTest()` and keep using it like before?

Unless you want to allow a plugin to override this value?

Reply:
Initially my thought was to make it flexible on where we store the `test` plugin binary. You brought a really good point here, considering we are following a very generic approach on where and how we are storing test plugin binary, I think we do not need to add `TestPluginInstallationPath`  in the `PluginDescriptor`. Also we can use `RunTest` function to determine the location. I will update the code based on this.